search-commons/src/test/java/no/unit/nva/search/ElasticsearchTest.java

```diff
@@ -118,10 +122,30 @@ void shouldReturnOneHitOnViewingScopeWithExcludedUnit() throws Exception {
 
         SearchResponse response = searchClient.findResourcesForOrganizationIds(INDEX_NAME, viewingScope);
 
-        MatcherAssert.assertThat(response.getHits().getHits().length,
+        assertThat(response.getHits().getHits().length,
                 is(equalTo(ONE_HIT_BECAUSE_ONE_UNIT_WAS_EXCLUDED)));
     }
 
+    @Test
+    void shouldReturnSearchResourcesResponse() throws Exception {
+        indexingClient.addDocumentToIndex(getIndexDocument(Set.of(INCLUDED_ORGANIZATION_ID)));
+        indexingClient.addDocumentToIndex(getIndexDocument(Set.of(INCLUDED_ORGANIZATION_ID)));
+
+        Thread.sleep(DELAY_AFTER_INDEXING);
+
+        ViewingScope viewingScope = getEmptyViewingScope();
```

Comment:
```var viewingScope=ViewingScope.create(INCLUDED_ORGANIZATION_ID);```

Reply:
What do you mean?

Reply:
replace lines 136-137 with the suggested line

Reply:
Done